packages/jest-cli/src/reporters/coverage_reporter.js

```diff
@@ -94,12 +94,8 @@ export default class CoverageReporter extends BaseReporter {
       }
 
       let coverageReporters = this._globalConfig.coverageReporters || [];
-      if (
-        !this._globalConfig.useStderr &&
-        coverageReporters.length &&
-        coverageReporters.indexOf('text') === -1
-      ) {
-        coverageReporters = coverageReporters.concat(['text-summary']);
+      if (!this._globalConfig.useStderr && coverageReporters.length < 1) {
+        coverageReporters = coverageReporters.push('text-summary');
```

Comment:
assignment is not necessary as push mutates the array

Reply:
Oops. Forgot to delete the assignment when I changed to push instead of concat